### PR TITLE
Tree coloring

### PR DESCRIPTION
**Pull request by albert-smit**
changes the coloring of a tree

### CODE REVIEW

**Comment by joswinter** [comment]
Effect looks nice, code looks good!

**Comment by jorenham** [comment]
Looks good, but see https://github.com/ProgrammingLife5/LifeTiles/issues/210

**Comment by rutgervandenberg** [comment]
This introduces a bunch of checkstyle violations.

**Comment by alangerak** [comment]
Looks good

File: lifetiles-tree/src/main/java/nl/tudelft/lifetiles/tree/view/SunburstRingSegment.java [hunk 1]
```diff
@@ -23,10 +23,13 @@
 
 public class SunburstRingSegment extends AbstractSunburstNode {
     /**
-     * the default color for this segment.
+     * the max brightness for the color of a Node.
      */
-    private static final Color DEFAULT_COLOR = Color.RED;
-
+    private static final double MAX_BRIGHTNESS = 0.8d;
```
Comment (jorenham) [inline lifetiles-tree/src/main/java/nl/tudelft/lifetiles/tree/view/SunburstRingSegment.java]: The `d` is redundant

File: lifetiles-tree/src/main/java/nl/tudelft/lifetiles/tree/view/SunburstRingSegment.java [hunk 2]
```diff
@@ -23,10 +23,13 @@
...
+    /**
+     * The default saturation of the color of a Node.
+     */
+    private static final double SATURATION = 0.9d;
```
Comment (jorenham) [inline lifetiles-tree/src/main/java/nl/tudelft/lifetiles/tree/view/SunburstRingSegment.java]: The `d` is redundant

File: lifetiles-tree/src/main/java/nl/tudelft/lifetiles/tree/view/SunburstRingSegment.java [hunk 3]
```diff
@@ -88,13 +88,15 @@ public SunburstRingSegment(final PhylogeneticTreeItem value,
...
 
-        result.setFill(createColor());
+        result.setFill(createColor(degreeStart, layer));
         result.setFillRule(FillRule.EVEN_ODD);
 
         // check if this is a large arc
-        double arcSize = AbstractSunburstNode.calculateAngle(degreeStart, degreeEnd);
+        double arcSize = AbstractSunburstNode.calculateAngle(degreeStart,
+                degreeEnd);
         boolean largeArc = arcSize > (AbstractSunburstNode.CIRCLEDEGREES / 2);
```
Comment (jorenham) [inline lifetiles-tree/src/main/java/nl/tudelft/lifetiles/tree/view/SunburstRingSegment.java]: redundant parentheses

**Pull request merged**